src/common/components/list-item/ListItem.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <li class="list-item">
+        <list-item-title class="title" :title="title"
+                         :search-input-text="inputText"
+                         :is-pinned="isPinned" :is-new="isNew"
+        />
+        <div class="info">
+            <p-badge outline :style-type="noticeTypeBadge.style">
+                {{ noticeTypeBadge.label }}
+            </p-badge><span>date</span><p-i width="0.125rem" name="ic_divider-dot" />
+            <span>writer</span><p-i width="0.125rem" name="ic_divider-dot" />
+            <span class="view-count"><p-i name="ic_view" width="1.125rem" /> 1,234</span>
+        </div>
+    </li>
+</template>
+
+<script lang="ts">
+import type { PropType } from '@vue/composition-api';
+import { computed, reactive, toRefs } from '@vue/composition-api';
+
+import { PBadge, PI } from '@spaceone/design-system';
+
+import { i18n } from '@/translations';
+
+import ListItemTitle from '@/common/components/list-item/ListItemTitle.vue';
+
+import type { NoticeType } from '@/services/my-page/notice/config';
+import { NOTICE_TYPE } from '@/services/my-page/notice/config';
+
+export default {
+    name: 'ListItem',
+    components: {
+        ListItemTitle,
+        PBadge,
+        PI,
+    },
+    props: {
+        title: {
+            type: String,
+            default: '',
+        },
+        noticeType: {
+            type: String as PropType<NoticeType>,
+            default: '',
+        },
+        inputText: {
+            type: String,
+            default: '',
+        },
+        isNew: {
+            type: Boolean,
+            default: false,
+        },
+        isPinned: {
+            type: Boolean,
+            default: false,
+        },
+    },
+    setup(props) {
+        const state = reactive({
+            // song-lang
+            noticeTypeBadge: computed(() => {
+                switch (props.noticeType) {
+                case NOTICE_TYPE.SYSTEM:
+                    return {
+                        label: i18n.t('시스템 공지'),
+                        style: 'primary',
+                    };
+                case NOTICE_TYPE.INTERNAL:
+                    return {
+                        label: i18n.t('내부 공지'),
+                        style: 'gray',
+                    };
+                default: return '';
+                }
+            }),
+        });
+        return {
+            ...toRefs(state),
+        };
+    },
+};
+</script>
+
+<style scoped lang="postcss">
+.list-item {
+    @apply border-b border-gray-200;
+    padding: 1rem;
+    &:hover {
+        @apply bg-blue-100;
+    }
```

Comment:
```suggestion
@media (hover: hover) {
    &:hover {
        @apply bg-blue-100;
    }
 }
```

이렇게 하면 더 좋을거같아요. 
안그러면 모바일에서 항상 호버상태가 보여집니다!

Reply:
이런 방법이 있군요! 감사합니다!

Reply:
헐 진짜요?!?! 저도 알아갑니다..